BookingsAPI/Bookings.Api.Contract/Responses/SuitabilityAnswersResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace Bookings.Api.Contract.Responses
+{
+    public class SuitabilityAnswersResponse : PagedCursorBasedResponse
+    {
+        /// <summary>
+        /// Gets or sets a list of participants suitability answers.
+        /// </summary>
+        public List<ParticipantSuitabilityAnswerResponse> ParticipantSuitabilityAnswerResponse { get; set; }
```

Comment:
Should be ParticipantSuitabilityAnswerResponses ? plural

Reply:
I am not sure, as return one response which is list